Ccna-automation-api-guide.md

```
@@ -0,0 +1,542 @@
+# CCNA Automation「APIの理解と活用」完全ガイド
+## 〜試験項目 2.0 Understanding and Using APIs をステップバイステップで攻略する〜
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)